src/Session.js

```diff
@@ -16,6 +17,7 @@ export const SessionProvider = ({ children }) => {
         const globalData = doc.data();
         if (globalData && globalData.activeSession) {
           setUpSessionListener(globalData.activeSession); // Gets updated from here
+          setNextSession(globalData.nextSession);
```

Comment:
well, i can't think of any usage for the whole session, we just need the id for the transition of users, and soon it will become the actual session and everything will be retrieved.

If condition added like you suggested.